specification/transport/udp/udp.tex

```diff
@@ -0,0 +1,370 @@
+
+\begin{remark}
+    Certain states provided in the header duplicate information that is already available in the IP header
+    or the multicast group address.
+    This is done for reasons of unification of the header format with other standard transport layer definitions,
+    and to simplify the access to the transfer parameters that otherwise would be hard to reach above the
+    network layer, such as the DSCP value.
+    The latter consideration is particularly important for forwarding nodes.
+\end{remark}
+
+\subsection{Transfer payload}
+
+After the transfer payload is constructed but before it is scheduled for transmission over the network,
+it is appended with the transfer CRC field.
+The transfer CRC function is \textbf{CRC-32C} (section~\ref{sec:appendix_crc32c}),
+and its value is serialized in the little-endian byte order.
+The transfer CRC function is applied to the entire transfer payload and only transfer payload.
+
+The transfer CRC is provided for all transfers,
+including single-frame transfers and transfers with an empty payload\footnote{%
+    This provides end-to-end integrity protection for the transfer payload.
+}.
+An implementation receiving a transfer should verify the correctness of its transfer CRC.
+
+\begin{remark}
+    From the perspective of the multi-frame segmentation logic, the transfer CRC field is part of the transfer payload.
+    From the definition of the header format it follows that the transfer CRC can only be found at the end of
+    the packet if the \verb|end_of_transfer| bit is set,
+    unless the transfer CRC field has spilled over to the next frame
+    (in which case the frame would contain only the transfer CRC itself or the tail thereof).
+\end{remark}
+
+\subsection{Maximum transmission unit}
+
+In this section, the maximum transmission unit (MTU) is defined as the maximum size of a UDP/IP datagram payload.
+
+This specification does not restrict the MTU of the underlying transport.
+It is recommended, however, to avoid MTU values less than 508~bytes,
+allowing applications to exchange up to $508 - 24 - 4 = 480$ bytes of payload in a single-frame transfer.
+Limiting the MTU at this value allows nodes that do not transmit and/or receive transfers larger than 480~bytes
+to omit support for multi-frame transfer decomposition and/or reassembly.
+
+Nodes emitting Cyphal/UDP traffic \emph{should} avoid IP fragmentation by
+limiting the size of the UDP payload accordingly\footnote{
+    Transfers larger than the MTU limit will be transmitted as multi-frame Cyphal transfers
+    (section~\ref{sec:transport_transfer_payload}).
+    The preference towards Cyphal framing over IP framing is to remove the limitation on the maximum transfer size
+    imposed by the UDP protocol and to permit preemption of long transfers by higher-priority transfers.
+    Further, the support for reassembly of large fragmented IP packets is not universal.
```

Comment:
Let's reference this RFC -> https://datatracker.ietf.org/doc/html/rfc8900 <- and just say "Cyphal does not require IP framing per the recommendations of RFC8900"